src/detectors/ECTRK/ECTRK.cc

```diff
@@ -21,7 +21,10 @@ void InitPlugin(JApplication *app) {
         "EndcapTrackerDigiHit",
         {"TrackerEndcapHits"},
         {"EndcapTrackerDigiHit"},
-        {}, // default config
+        {
+            .threshold = 0.65 * dd4hep::keV,
+            // .timeResolution = 10,
```

Comment:
```suggestion
```

Reply:
I didn't see any field that said ten, but I'll put it in.

Reply:
I believe, the suggestion was to remove the commented lines (keeping timeResolution at 0).

Reply:
Yes, though the default is 10, I think. (Not easy to verify right now.)

Reply:
It's 8 actually, for no discernible reason.
https://github.com/eic/EICrecon/blob/main/src/algorithms/digi/SiliconTrackerDigiConfig.h